server/issue.go

```diff
@@ -85,40 +130,23 @@ func decodePostActionRequest(r *http.Request) (string, model.PostActionIntegrati
 	return authenticatedUserID, requestData, 0, nil
 }
 
-func (p *Plugin) buildPostActionContext(authenticatedUserID string, requestData model.PostActionIntegrationRequest) (*postActionContext, int, string) {
+func (p *Plugin) buildPostActionContext(authenticatedUserID string, requestData model.PostActionIntegrationRequest, requireStoredPost bool) (*postActionContext, int, string) {
```

Comment:
Do we need this new parameter `requireStoredPost`? It seems that both instances that it's used it's `false`?